.github/workflows/build.yaml

```diff
@@ -46,10 +61,45 @@ jobs:
         PYTHON: python
     - name: Install dependencies
       run: make setup
+    - name: Create changelog and bump
+      if: ${{ github.ref == 'refs/heads/main' }}
+      run: |
+        git config --global user.name "Action User"
+        git config --global user.email "action@github.com"
+        git config --list --global
+        cz changelog
+        cz bump
     - name: Build the package
-      # We don't need to check and test the package separately
-      # because `make dist` runs those targets first and only
-      # builds the package if they succeed.
-      run: make dist
-    - name: Build the docs
-      run: make docs
+      # We don't need to check and test the package separately because `make dist` runs
+      # those targets first and only builds the package if they succeed.
+      run: |
+        make dist docs requirements sbom
+    - name: Compute package hash
+      if: ${{ matrix.python == '3.10' && matrix.os == 'ubuntu-latest' }}
+      id: compute-hash
+      shell: bash
+      run: |
+        set -euo pipefail
+        TARBALL_PATH=$(find ${{ env.GENERATED_PACKAGE_DIR }} -name "*.tar.gz")
+        WHEEL_PATH=$(find ${{ env.GENERATED_PACKAGE_DIR }} -name "*.whl")
+        SBOM_PATH=$(find ${{ env.GENERATED_PACKAGE_DIR }} -name "*-sbom.json")
+        HTML_DOCS_PATH=$(find ${{ env.GENERATED_PACKAGE_DIR }} -name *-docs-html.zip)
+        DIGEST=$(sha256sum $TARBALL_PATH $WHEEL_PATH $SBOM_PATH $HTML_DOCS_PATH | base64 -w0)
+        echo "digest of artifacts is $DIGEST."
+        echo "::set-output name=artifacts-sha256::$DIGEST"
+    - name: Upload the package artifact
+      # For now only generate artifacts for Python 3.10 on ubuntu-latest.
+      # Currently reusable workflows do not support setting strategy property from the caller workflow.
+      # See: https://docs.github.com/en/actions/using-workflows/reusing-workflows#limitations.
```

Comment:
Maybe hoist this comment to the top of the file? We’ve mentioned this up there too, I think.